services/src/main/java/org/keycloak/connections/httpclient/ProxyMappingsAwareRoutePlanner.java

```diff
@@ -45,9 +50,16 @@ public ProxyMappingsAwareRoutePlanner(ProxyMappings proxyMappings) {
   @Override
   protected HttpHost determineProxy(HttpHost target, HttpRequest request, HttpContext context) throws HttpException {
 
-    HttpHost proxy = proxyMappings.getProxyFor(target.getHostName());
-    LOG.debugf("Returning proxy=%s for targetHost=%s", proxy, target.getHostName());
-
+    String targetHostName = target.getHostName();
+    HttpHost proxy = proxyMappings.getProxyFor(targetHostName);
+    String[] proxyCredentials = proxyMappings.getProxyCredentialsFor(targetHostName);
+    LOG.debugf("Returning proxy=%s for targetHost=%s", proxy, targetHostName);
+    if (proxyCredentials != null && proxyCredentials.length == 2) {
+      NTCredentials ntCreds = new NTCredentials(proxyCredentials[0], proxyCredentials[1], null, null);
```

Comment:
Please use 4 space indentation
```suggestion
        NTCredentials ntCreds = new NTCredentials(proxyCredentials[0], proxyCredentials[1], null, null);
```

Reply:
Two spaces and four spaces for indentation are mixed in `ProxyMappings.java` and `ProxyMappingsAwareRoutePlanner.java `. Should I convert all of the former to the latter?